### ISSUE TITLE
(tlg0007.tlg068.perseus-grc1.xml) EpiDoc and CTS conversion 

### ISSUE THREAD

**Issue by ahanhardt** [PerseusDL/canonical-greekLit#583]
Claiming file for conversion work

**Comment by ahanhardt** [comment]
@lcerrato Please bump the URN for file tlg0007.tlg068.perseus-grc1.xml

**Comment by lcerrato** [comment]
Hi @ahanhardt 
tlg0007.tlg068.perseus-grc1.xml bumped to tlg0007.tlg068.perseus-grc2.xml

**Comment by lcerrato** [comment]
Hi @ahanhardt @Ethan-DR

Will there be more files forthcoming on this? The PR indicates a fix. If you want this to be a fix, then we should change the title of this issue to reflect only the work on the CTS work file (as opposed to a complete conversion of all files). 

I've made some comments on the PR. Looks great.

**Comment by ahanhardt** [comment]
Hi Lisa, 

Yes, we have the Greek file ready to be pushed, we were just waiting for the CTS file to be accepted.  The fix comment was my bad, sorry!

**Comment by lcerrato** [comment]
@ahanhardt 
cts file looks good! I won't merge the PR then.

**Comment by ahanhardt** [comment]
Logistical question for you: another intern has the Greek file ready to be pushed, so should we wait until Ethan’s PR is merged, align the other intern’s repo, then send in the new PR with the edited Greek file? Or just push the other intern’s edited Greek file and repo now?

**Comment by lcerrato** [comment]
@ahanhardt 
In my own workflow I complete all of the related files in the same issue.
For external contributions, we recommend one file per issue and per pull request.

I just merged the work file changes but I did not list this issue as fixed (so that we can keep it open).

So subsequent pull requests should all refer to this issue number but let's do one pull request per file.

When it's all set, I'll fix the issue with the last pull request.

Thanks!
